abstract_specification.md

```diff
@@ -1,31 +1,27 @@
 # Introduction
 
-> **Status**: in active development. Version: `0.2.0-dev`
->
-> ℹ️ **Note** - this is an early working draft (WORK IN PROGRESS). It may contain controversial, undecided, or incomplete proposals - feedback is encouraged. It is expect that additional revisions will be required to converge the underlying primitives with use cases, and to incorporate lessons from initial implementation and early users. Please see `issues` and other call-outs throughout the doc for a list of active discussions/debates.
```

Comment:
I believe this is subsumed by GitHub versioning. (Also this is long out of sync with the spec repo's release tag.)

Reply:
The versioning of the specification and the versioning of Python releases are not the same, so I'm not sure whether this should be deleted.

Reply:
@bkmartinjr ? I don't recall seeing `0.2.0-dev` anywhere else, & I _suspect_ this is stale -- but, if this word `0.2.0-dev` _right here_ is the very place where it _is_ important -- I should know that.

Happy to go either way; please lmk your thoughts here.

Reply:
@thetorpedodog - you are correct.  The spec version (returned by `get_SOMA_version`) is defined by this line.

I would be happy if we incremented it at this point in time.

I think the `Note` could safely be replaced with a comment that indicates where & how to send input / feedback, rather than the blanket caveat it currently expresses.

Reply:
And @johnkerl - our current `tiledbsoma` API `get_SOMA_version` is _supposed_ to return this string. I will go ensure it does.

Reply:
Current Python implementation is as expected:
```
def get_SOMA_version() -> str:
    """
    Return semver-compatible version of the supported SOMA API.

    [lifecycle: experimental]
    """
    return "0.2.0-dev"
```